### PR TITLE
Fixed #34761 -- Dropped support for MySQL < 8.0.11.

### PR DESCRIPTION
**Pull request by felixxm**
ticket-34761

### CODE REVIEW

**Review by ngnpope — APPROVED** [review]
LGTM. One minor tweak.

File: django/db/backends/mysql/features.py [hunk 1]
```diff
@@ -255,20 +247,17 @@ def can_introspect_check_constraints(self):
...
             return self.connection.mysql_version >= (10, 6)
-        return self.connection.mysql_version >= (8, 0, 1)
+        return True
 
     @cached_property
     def has_select_for_update_nowait(self):
         if self.connection.mysql_is_mariadb:
             return True
-        return self.connection.mysql_version >= (8, 0, 1)
+        return True
```
[the remark this answers is not in the document]
Comment (felixxm) [reply django/db/backends/mysql/features.py:256]: Good catch, updated :+1:

**Comment by felixxm** [comment]
@ngnpope Thanks for checking :+1:

**Review by nessita — APPROVED** [review]
LGTM

**Comment by felixxm** [comment]
Thanks both for checking :+1:

**Pull request merged**